autoarray/inversion/pixelizations.py

```diff
@@ -132,6 +132,7 @@ def mapper_from_grid_and_sparse_grid(
         self,
         grid: grids.Grid,
         sparse_grid: grids.Grid = None,
+        sparse_image_plane_grid: grids.Grid = None,

```

Comment:
If the default value is None then the type should be Optional[grids.Grid]

Reply:
Agreed.